go/mysql/slave_status.go

```diff
@@ -23,13 +23,20 @@ import (
 
 // SlaveStatus holds replication information from SHOW SLAVE STATUS.
 type SlaveStatus struct {
-	Position            Position
-	SlaveIORunning      bool
-	SlaveSQLRunning     bool
-	SecondsBehindMaster uint
-	MasterHost          string
-	MasterPort          int
-	MasterConnectRetry  int
+	Position Position
+	// RelayLogPosition is the Position that the replica would be at if it
+	// were to finish executing everything that's currently in its relay log.
+	// However, some MySQL flavors don't expose this information,
+	// in which case RelayLogPosition.IsZero() will be true.
+	RelayLogPosition     Position
+	FilePosition         Position
+	FileRelayLogPosition Position
+	SlaveIORunning       bool
+	SlaveSQLRunning      bool
+	SecondsBehindMaster  uint
+	MasterHost           string
```

Comment:
The proto generation here made it `MasterServerId`. Is there a way to force `ID` to be in all caps? Otherwise I think it makes more ense to make it `MasterServerId` in the `SlaveStatus` struct as well so they match. Thoughts?

Reply:
I ended up changing it to `MasterServerID` in `SlaveStatus` and leaving it as the auto-generated `MasterServerId` in the `Status` struct. LMK what you think.